incubator/haproxy-ingress/templates/controller-daemonset.yaml

```diff
@@ -148,10 +156,10 @@ spec:
             - name: metrics
               containerPort: 9101
               protocol: TCP
-          readinessProbe:
+          livenessProbe:
             httpGet:
               path: /
-              port: 9101
+              port: metrics
```

Comment:
Is this right here ?  any document

Reply:
This is referring to the container port by name instead of number. Otherwise functionally identical.